examples/simple/posix/UB-PTHREAD2-bad.output

```diff
@@ -0,0 +1,12 @@
+Error: UB-PTHREAD2
+Description: Destroying a locked mutex.
+Locked here:
+  at pthread_mutex_lock(UB-PTHREAD2-bad.c:15)
+  at locker_thread(UB-PTHREAD2-bad.c:15)
+  at <file-scope>(<unknown>)
+
```

Comment:
Is this white line necessary?  I initially thought that there are two error messages.  Others may be confused, too.